consensus/state.go

```diff
@@ -1597,7 +1607,9 @@ func (cs *ConsensusState) addVote(vote *types.Vote, peerID p2p.ID) (added bool,
 		blockID, ok := precommits.TwoThirdsMajority()
 		if ok {
 			if len(blockID.Hash) == 0 {
-				cs.enterNewRound(height, vote.Round+1)
+				cs.enterNewRound(height, vote.Round)
```

Comment:
This code does not seem to be related to delaying validator set changes?

Reply:
Sure, I'll pull the consensus liveness fix into a separate PR.